demos/using-dev-build/README.md

```diff
@@ -1,9 +1,17 @@
+
+[comment]: # This document is mainly written to be viewed on https://internetcomputer.org/docs/current/developer-docs/integrations/internet-identity/integrate-identity/
+
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
+This guide shows how to integrate and test a project with Internet Identity. The complete code example can be found [here](https://github.com/dfinity/internet-identity/tree/main/demos/using-dev-build). It uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
 This is a standalone project that you can copy to your own project.
 
+## Prerequisites
```

Comment:
Nice! how about also adding prerequisites about what the user should know already? i.e. "this assumes you know what a [canister](https://whatisacanister.foo) is, how to use `dfx` to deploy a canister, etc,etc"

Reply:
Good idea! I'll add that.